File: mssp/archaeology/012-cpython-dbm/src/SCL/policy.py
"""Which schedule this deployment runs, and what it refuses to let pass."""
import json
import pathlib

_C = json.loads((pathlib.Path(__file__).parent / "policy.json").read_text(encoding="utf-8"))

schedule = lambda: _C["schedule"]                                                    # noqa: E731
declarations_must_match_behaviour = lambda: bool(_C["declarations_must_match_behaviour"])  # noqa: E731

File: mssp/examples/012-two-writers/src/SCL/policy.mjs
// Which medium and operation this deployment runs, and what it refuses to ship.
import fs from "node:fs";
import path from "node:path";
import { fileURLToPath } from "node:url";

const here = path.dirname(fileURLToPath(import.meta.url));
const config = JSON.parse(fs.readFileSync(path.join(here, "policy.json"), "utf8"));

export const medium = () => config.medium;
export const operation = () => config.operation;
export const unmetIsFatal = () => Boolean(config.unmet_requirement_is_fatal);
